### DISCUSSION TITLE
Logo Mapant ES

### DISCUSSION THREAD

**Discussion by manudona92** [manudona92/MapantES#7] [Ideas]
Abro primera discusión y para tenerla registrada. 

Si en un futuro queremos un logo, estaba pensando en como podria ser...

![image](https://user-images.githubusercontent.com/76484741/103442323-6f68c380-4c55-11eb-95d2-9891f86917de.png)

Donde cada hormiga porte una hoja, la primera con la forma de España península, la segunda con un mordisco de Baleares y la tercera con un mordisco con la forma de Canarias. Otra opción sería una única hormiga con la forma mordisqueada de España península para simplificar.

**Comment by jarufe** [comment]
Como banner o imagen grande que acompañe a titulares, documentos o páginas web me parece bien ese estilo de imagen que propones. Sin embargo, para logo tal vez habría que buscar algo más esquemático y simple, ¿no?. Por ejemplo, una hormiga dejando un rastro (para indicar que va caminando por) por la sobreimpresión de la península

**Reply by manudona92** [reply]
Estoy de acuerdo, para logo tiene que ser algo simple. Por eso habia pensado en de una única hormiga llevando una hoja con la forma de la península. 
Intento imaginas lo que tu dices, pero no lo asocio bien ¿Podrias hacer un "paint" rápido?. No corre prisa. Tengo algun colega que sabe de diseño que quizás podría echar un cable con las ideas que le ofrecemos

**Reply by jarufe** [reply]
![MapAnt_Logo1](https://user-images.githubusercontent.com/17064412/103463581-b9bd7380-4d2d-11eb-89f2-e5f2ea931d2c.png)
![MapAnt_Logo2](https://user-images.githubusercontent.com/17064412/103463585-c3df7200-4d2d-11eb-9750-ea7606fea7ad.png)
![MapAnt_Logo3](https://user-images.githubusercontent.com/17064412/103463588-c8a42600-4d2d-11eb-83a4-d391efaa16c6.png)

La idea podría ser alguna de las anteriores. Podrían ser 2 en vez de 3 hormigas, un poco más grandes, en distintas direcciones. La última imagen pretende indicar que cada hoja en lugar de hoja es un trozo de cuadrícula ya elaborada que se va a colocar en algún lugar del mapa.

**Comment by manudona92** [comment]
Adjunto por aquí el proto logo de Mapant ESP. Es simplemente el Hierro, maquetando un poco el agua. 

Cogí el Hierro porque ha sido nuestra isla test, y que le hemos cogido cariño...Lo dicho, cualquier otra idea que se nos o les ocurra a otro volutario el dia de mañana la incluimos, ya que esto es una imagen con mucho detalle (la utilizaría de foto de perfil para el grupo de facebook y twitter)

![Hierro_todo_reducido](https://user-images.githubusercontent.com/76484741/110243698-be83eb00-7f5b-11eb-9fde-af658b606b49.jpg)
